data/wanderer/wanderers middle.txt

```diff
@@ -3087,6 +3085,9 @@ mission "Wanderers: Sestor: Exiles 3"
 					goto end
 				`	"What are they asking for in return?"`
 			`	Rek says, "They request a meeting with our leaders, to work out terms of trade for our mutual benefit. It is something we would certainly agree to with no difficulty."`
+			action
+				log "Succeeded in communicating with the Korath exiles, by landing on the one habitable planet in their territory. (It seems like the exiles might have a taboo against using weapons on that world.) The exiles agreed to send some ships to help disable the remaining Kor Sestor factory."
+				event "wanderers: exiles peaceful"
```

Comment:
This won't fire if you jump straight to the end by choosing "That sounds reasonable."

I don't see any reason why these would necessarily need to be a conversation action. I think it'd be cleaner to move everything to an `on accept`.